ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,42 @@
+from pyrsistent import b
+
+bool_to_int = lambda value: int(value)
+print(bool_to_int(True))
+print(bool_to_int(False))
+
+get_smaller = lambda a,b: min (a,b)
+print ("get smaller:", (get_smaller(16,31)))
+print ("get smaller:", (get_smaller(253,223)))
+
+def cube (base):
+  return (base**3)
+print  (cube(3))
+print  (cube(5))
+
+def absolute_difference (a, b) :
+  return abs ( a,b )
+print ("absolute difference:", (abs (14-11)))
+print ("absolue difference:", (abs (13-40)))
```

Comment:
Please call the `absolute_difference` function to get these values. Make sure your `absolute_difference` function definition is using `abs` correctly: https://www.w3schools.com/python/ref_func_abs.asp